bridgepoint/schema.py

```diff
@@ -17,7 +17,7 @@
 # License along with pyxtuml. If not, see <http://www.gnu.org/licenses/>.
 
 
-__version__ = 5.2
+__version__ = 5.3
```

Comment:
The version number I picked previously was based on the bridgepoint editor version from which the schema was first introduced (i.e. currently bridgepoint 5.2). I would bump this to the current version

Reply:
done